integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -715,46 +724,13 @@ public void tearDownAll() {
           .withFailMessage("uninstallNginx() did not return true")
           .isTrue();
     }
-
-    // shutdown domain1
-    logger.info("Shutting down domain1");
-    assertTrue(shutdownDomain(domain1Uid, domain1Namespace),
-            String.format("shutdown domain %s in namespace %s failed", domain1Uid, domain1Namespace));
-
     // delete mii domain images created for parameterized test
```

Comment:
Should we check if SKIP_CLEANUP is set to true, we should avoid the clean up for a debug session,

Reply:
this should be done in cleanutil method, I just removed unnecessary delete in here

Reply:
this step need to be done before 

> Looks good. Update the PR with Jenkin Result

results are in description, I have updated with latest today